modules/core/04-channel/types/codec.go

```diff
@@ -6,6 +6,7 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/msgservice"
 
+	v2types "github.com/cosmos/ibc-go/v9/modules/core/04-channel/v2/types"
```

Comment:
hm, I don't think we should be importing v2Types from v1, I think we should have a separate 04-channel/v2/types/codec.go WDYT?

Reply:
Think it would be great to create a clear separation

Reply:
I think you are right :)